src/cript/nodes/util/__init__.py

```diff
@@ -37,6 +38,14 @@ def default(self, obj):
                 if uid in NodeEncoder.handled_ids:
                     return {"node": obj._json_attrs.node, "uid": uid}
 
+            try:
+                uuid_str = str(obj.uuid)
+            except AttributeError:
```

Comment:
why are we catching an error and not doing anything with it?

Reply:
yes, that is a very pythonic way.

Instead of testing if an attribute is present, this is preferred.

Reply:
I agree, the principle of `try instead of checking for if`